purview/api/v1.py

```diff
@@ -80,19 +81,18 @@ def get_resource_story(resource, uid):
     if backward_query:
         results_unordered.update(query_neo4j(backward_query))
 
-    results = []
+    results = OrderedDict({'data': [], 'meta': {}})
     curr_label = 'BugzillaBug'
     while curr_label:
         if curr_label in results_unordered:
-            # Have each index of results be a list with the 0 index being the resource name
-            # and the the rest of the indexes being resources
-            results_unordered[curr_label].insert(0, curr_label)
-            results.append(results_unordered[curr_label])
+            results['data'].append(results_unordered[curr_label][0])
 
         curr_label = story_flow[curr_label]['forward_label']
 
-    if len(results) == 0:
+    if len(results['data']) == 0:
         # If no relationships are found, return the artifact by itself
-        results.append([item.__label__, item.serialized])
+        results['data'].append(item.serialized)
```

Comment:
This needs to have the `resource_type` key defined on it.

Reply:
It already does

Reply:
I don't think so since you add that key in `query_neo4j` and not `serialized`. That is an idea though, you could add the key directly in `serialized`.